plugin/pkg/scheduler/factory/factory.go

```diff
@@ -379,6 +400,17 @@ func (c *configFactory) onPvcAdd(obj interface{}) {
 	}
 }
 
+func (c *configFactory) onPvcUpdate(oldObj interface{}, obj interface{}) {
```

Comment:
This is not right.  Need to also think about how this works when invalidating the predicates after assume.

Reply:
If we always invalidate after assumeVolume, then no need to invalidate on pvc update, so this can be removed.